### PR TITLE
⚰️ Remove dead code

### PR DESCRIPTION
**Pull request by josemigallas**
* [DashboardHelper](https://github.com/3scale/porta/blob/HEAD/app/helpers/provider/admin/cms/dashboard_helper.rb)
* [PartnersHelper](https://github.com/3scale/porta/blob/HEAD/app/helpers/whitelabel/admin/accounts/partners_helper.rb)

There is no occurrence of any of the methods contained in this module. It's safe to remove.

* [results.html.erb](https://github.com/3scale/porta/blob/25fa9bc53bdd3eeb8f2a9c8e882f771c85c3c3cb/app/views/buyer/search/results.html.erb)

This template reference partials that were long ago removed, e.g. [buyer/accounts/_accounts.html.erb](https://github.com/3scale/system/commit/cdbc6d2c61680c724574f4d6577c7880ef341024#diff-4a09223936379874acc240ecacfcf1bc14f6846d06ecfc1d799ff399db0692cb) so it can be safely removed.

* app/views/accounts

The only reference to this dir is the previous template. None of the partial inside [accounts/form](https://github.com/3scale/porta/blob/HEAD/app/views/accounts/accounts/_form.html.erb), [invitations/form](https://github.com/3scale/porta/blob/f4ca0f14168e6852f323ddd6f5d7fd9d2e85a311/app/views/accounts/invitations/_form.html.erb) and is [profiles/form](https://github.com/3scale/porta/blob/f4ca0f14168e6852f323ddd6f5d7fd9d2e85a311/app/views/accounts/invitations/_form.html.erb) referenced anywhere else, so they are safe to remove.

* [form/settings](https://github.com/3scale/system/blob/b9749d3071c0fbf46979d786977eb768198b35f7/app/views/api/services/forms/_settings.html.slim)

This partial stopped being used here https://github.com/3scale/porta/commit/ad2cf0f89b031b2b35bcd48fd721784a530f98f4#diff-7c1a2308d6dc4091c7caf1cff22a79244810b3218c7a456f2ef43a2181651ffeL20 but somehow got out of the purge. Not anymore. Its content is duplicated in [forms/usage_rules](https://github.com/3scale/porta/blob/c0040ca9d08524d4b6bdc0c9192e492789f20ff9/app/views/api/services/forms/_usage_rules.html.slim).

* [latest_signups](https://github.com/3scale/porta/blob/HEAD/app/views/buyers/applications/_latest_signups.html.erb)
* [accounts_pending](https://github.com/3scale/porta/blob/HEAD/app/views/buyer/accounts/_accounts_pending.html.erb)
* [service_navigations](https://github.com/3scale/porta/blob/62a951e6f9929cc3f0d782deb2e96601309576cd/app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim)

This partials are not referenced anywhere. Safe to remove.

* Old vertical nav partials (https://github.com/3scale/porta/blob/HEAD/app/views/shared/provider/navigation)

The nav is now rendered by React. In fact these partials stopped being used a long time ago. Safe to remove.

* ⚠️ [apidocs/base/show](https://github.com/3scale/porta/blob/b38ca6c85353c0a01ef9689f4b188adb198abad3/app/views/admin/api_docs/base/active_docs.html.erb)

Not sure why this template exists and when it stopped being used but I haven't found any reference to it and I haven't been able to navigate to it. Instead of "show", what api_docs use is "preview". Trying to access apidocs/show via URL returns 404.

* [applications/applications/show]()

This template was originally removed about [9 years ago](https://github.com/3scale/system/commit/0497b0c6d840342857416c214c1943671b01baaa#diff-7e133f33fbaff4dfab10132e9e18f5708b059c26f4ccc1b975aad4cf4475d7bb) and then [restored](https://github.com/3scale/system/commit/e6e61335bab2586f8020556f98db0e9b15be7ddc#diff-7e133f33fbaff4dfab10132e9e18f5708b059c26f4ccc1b975aad4cf4475d7bb) and it refers to "Dev portal files". I guess this was supposed to be moved to dev portal and eventually got forgotten and not used. It should be safe to remove.

* ⚠️ [services/stats](https://github.com/3scale/porta/blob/HEAD/app/views/api/services/stats.html.erb)

I can't find any route or reference to this template and there are no tests using it. I think it can be removed.

### CODE REVIEW

**Review by jlledom — APPROVED** [review]

**Comment by josemigallas** [comment]
> How can you be sure it's dead code?

We can't be 100% but I haven't found any indication of these files being reachable/in use.

File: app/views/provider/admin/destroys/index.html.erb [hunk 1]
```diff
@@ -1,25 +0,0 @@
-<% content_for :page_header_title, "Deleted #{@kind.human.pluralize}" %>
```
Comment (mayorova) [inline app/views/provider/admin/destroys/index.html.erb:1]: I wasn't aware of this feature, to be honest. But now when I see it, maybe it's actually useful :P We can as @3scale/support if they use it.

But well, if we decide to delete it, I'd say we also need to cleanup routes and the controller.

Comment (josemigallas) [reply app/views/provider/admin/destroys/index.html.erb:1]: Yeah you're right it probably deserves it's own PR

**Comment by josemigallas** [comment]
@jlledom @mayorova I investigated a bit about every file, found some more and re-made the commit history from scratch. Hopefully it's now safer to remove all these.. except there are a couple I haven't been able to 100% justify.

**Review by jlledom — APPROVED** [review]

**Review by mayorova — APPROVED** [review]
Nice!

**Pull request merged**